streams/src/test/java/org/apache/kafka/streams/integration/KTableKTableForeignKeyJoinIntegrationTest.java

```diff
@@ -92,8 +92,6 @@ public KTableKTableForeignKeyJoinIntegrationTest(final boolean leftJoin,
     public void before() {
         final String safeTestName = safeUniqueTestName(getClass(), testName);
```

Comment:
Is this method ```safeUniqueTestName``` useless now? It seems to me it tries to produce different name for each test case. Is it possible to make those tests flaky?

Reply:
Nice catch! @chia7712 , it's unused and I've removed this line now. Thanks.